top500_plot.py

```diff
@@ -18,12 +18,15 @@
 
 # get the data
 df = pd.read_csv('TOP500_history.csv', low_memory=False, parse_dates={'Date': ['Year','Month','Day']})
-assert (df.groupby(('Date',)).size()==500).all()
+assert (df.groupby(('Date')).size()==500).all()
```

Comment:
I'm guessing these worked with Pandas v0.17 (https://github.com/dlenski/top500/blob/master/requirements.txt#L3), which is now very ancient…

What version are you using, exactly? Can you make a corresponding edit to `requirements.txt` so that a compatible version of Pandas is required?

Reply:
I'm not sure since I did all of this work out of the /tmp/ directory since I just wanted to update the wikipedia image :)

But I ran the repo's instructions on Ubuntu 20.04 and found the script to not work without these changes